awx/ui_next/src/screens/Dashboard/shared/LineChart.jsx

```diff
@@ -197,61 +197,68 @@ function LineChart({ id, data, height, pageContext }) {
       vertical.transition().style('opacity', 0);
     };
 
-    // Add the successLine path.
-    svg
-      .append('path')
-      .data([formattedData])
-      .attr('class', 'line')
-      .style('fill', 'none')
-      .style('stroke', () => colors(1))
-      .attr('stroke-width', 2)
-      .attr('d', successLine)
-      .call(transition);
-
-    // Add the failLine path.
-    svg
-      .append('path')
-      .data([formattedData])
-      .attr('class', 'line')
-      .style('fill', 'none')
-      .style('stroke', () => colors(0))
-      .attr('stroke-width', 2)
-      .attr('d', failLine)
-      .call(transition);
-
     const dateFormat = d3.timeFormat('%-m-%-d');
 
-    // create our successLine circles
-    svg
-      .selectAll('dot')
-      .data(formattedData)
-      .enter()
-      .append('circle')
-      .attr('r', 3)
-      .style('stroke', () => colors(1))
-      .style('fill', () => colors(1))
-      .attr('cx', d => x(d.DATE))
-      .attr('cy', d => y(d.RAN))
-      .attr('id', d => `success-dot-${dateFormat(d.DATE)}`)
-      .on('mouseover', handleMouseOver)
-      .on('mousemove', handleMouseMove)
-      .on('mouseout', handleMouseOut);
-    // create our failLine circles
-    svg
-      .selectAll('dot')
-      .data(formattedData)
-      .enter()
-      .append('circle')
-      .attr('r', 3)
-      .style('stroke', () => colors(0))
-      .style('fill', () => colors(0))
-      .attr('cx', d => x(d.DATE))
-      .attr('cy', d => y(d.FAIL))
-      .attr('id', d => `fail-dot-${dateFormat(d.DATE)}`)
-      .on('mouseover', handleMouseOver)
-      .on('mousemove', handleMouseMove)
-      .on('mouseout', handleMouseOut);
-  }, [data, height, id]);
+    if (jobStatus !== 'failed') {
+      // Add the fail line path.
+      svg
```

Comment:
Maybe I'm misreading or reading too much into the comment, but should these conditionals be `if (jobStatus === 'failed')` instead of `!== 'failed')` ?

Reply:
yep, the comment is backwards.  should be `Add the success line`.